backend/src/routes/account.ts

```diff
@@ -284,16 +283,36 @@ router.post("/account-edit", async function (req, res) {
       null,
       (err, res) => {}
     );
-    if (newPassword.length > 0 && newPassword.trim().length > 0)
-     { 
-       user?.updateOne({ password: hash }, null, (err, res) => {});
-       return res.status(200).json({ message: "success with password change" });
-     }
+    if (newPassword.length > 0 && newPassword.trim().length > 0) {
+      user?.updateOne({ password: hash }, null, (err, res) => {});
+      return res.status(200).json({ message: "success with password change" });
+    }
     return res.status(200).json({ message: "success" });
   } catch (error) {
     return res.status(500).json({
       message: error.message,
     });
   }
 });
+
+router.post("/getuser", async function (req, res) {
+  let { token } = req.body;
+  let data = jwt.decode(token) as any;
+  let userId = data.userId;
+
+  try {
+    let user = await User.findOne({ _id: userId });
+
+    return res
+      .status(200)
+      .json({
+        user,
+        message: "success",
+      });
+  } catch (error) {
+    return res.status(500).json({
+      message: error.message,
+    });
+  }
+});
```

Comment:
Why is this here?

Reply:
just remove your changes from this file.

Reply:
I removed it from account